src/loaders/index.js

```diff
@@ -36,9 +36,9 @@ export async function createZarrLoader({
 /**
  * This function wraps creating a ome-tiff loader.
  * @param {Object} args
- * @param {String} props.url URL from which to fetch the tiff.
- * @param {Array} props.offsets List of IFD offsets.
- * @param {Object} props.headers Object containing headers to be passed to all fetch requests.
+ * @param {String} args.url URL from which to fetch the tiff.
+ * @param {Array} args.offsets List of IFD offsets.
+ * @param {Object} args.headers Object containing headers to be passed to all fetch requests.
```

Comment:
is this a typical convention? Just make sure to be consistent, which i think we are

Reply:
I'm not sure there's a typical convention.  We seem to be fairly consistent about calling React/DeckGL object-arguments `props` and otherwise using `args` as far as I can see.